src/common/libschedutil/init.c

```diff
@@ -0,0 +1,35 @@
+/************************************************************\
+ * Copyright 2019 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#include <flux/core.h>
+
+#include "init.h"
+
+schedutil_t *schedutil_create (flux_t *h)
+{
+    schedutil_t *s = calloc(1, sizeof(*s));
+    if (!s)
+        return NULL;
+
+    s->h = h;
+
+    return s;
+}
+
+void schedutil_destroy (schedutil_t* s)
+{
+    /*
```

Comment:
drop commented out code

Reply:
Whoops.  I dropped that in a commit somewhere but should have squashed/fixedup.  I'll make sure that gets out.  Thanks!